### ISSUE TITLE
금융공학 데이터 모델

### ISSUE THREAD

**Issue by light-src** [light-src/toberich#40]
- 프랜치
https://mba.tuck.dartmouth.edu/pages/faculty/ken.french/data_library.html
https://darktrader.tistory.com/entry/%ED%8C%8C%EB%A7%88-%ED%94%84%EB%A0%8C%EC%B9%98%EC%9D%98-3%EC%9A%94%EC%9D%B8-%EB%AA%A8%ED%98%95
- 블로그
https://psystat.tistory.com/147

**Comment by light-src** [comment]
CAPM, 3 Factor, 5 Factor

**Comment by changwoo-seo94** [comment]
5팩터 모형도 결국 결론은 '적정가치 = 1기현금흐름/할인율 + 2기현금흐름/할인율 + ... '으로 계산하게 되는데 분자에 있는 현금흐름을 기존 방식보다 더 현실화하기 위해서 현금배당에 대한 회귀분석으로 추정하는 방식으로 하는 것도 가능할까??(배당할인모형)

**Comment by jungHWANS** [comment]
일단 이모지 눈알 큰걸로 박아놨다

**Comment by jungHWANS** [comment]
네, 현금 배당과 관련한 회귀분석 방법이 있습니다. [김성민 교수님과 장용원 교수님께서 2017년에 발표한 "이익의 지속성에 근거한 배당정책과 기업가치"라는 논문에서는 이익의 지속성을 고려한 현금배당이 기업가치에 미치는 영향을 분석하였습니다](https://m.blog.naver.com/edamtax9/223029193624)[1](https://m.blog.naver.com/edamtax9/223029193624). [이 논문에서는 지속적인 이익에 근거한 현금배당이 기업가치에 미치는 영향을 분석하였으며, 실증분석 결과, 총자산 대비 현금배당 비중(D/A)이 증가할수록 기업가치는 증가하였으나 현재 이익 대비 현금배당 비중(D/E)이 증가할수록 기업가치는 증가하지 않았다는 결과를 얻었습니다](https://m.blog.naver.com/edamtax9/223029193624)[1](https://m.blog.naver.com/edamtax9/223029193624). [이 논문에서는 지속적인 이익 대비 현금배당 비중(D/EPERM)이 증가할수록 기업가치는 유의적으로 증가하였다는 결과도 얻었습니다](https://m.blog.naver.com/edamtax9/223029193624)[1](https://m.blog.naver.com/edamtax9/223029193624). [이러한 결과들은 국내 기업들에게 이익의 지속성을 고려한 최적배당정책이 기업가치 제고에 필요하다는 것을 보여주며, 기관투자자들에게는 적정배당수준에 대한 모니터링 역할이 주주가치 향상에 기여할 수 있음을 시사합니다](https://m.blog.naver.com/edamtax9/223029193624)[1](https://m.blog.naver.com/edamtax9/223029193624).

gpt 대답내용

**Comment by jungHWANS** [comment]
이거 논문 읽어보겠음

**Comment by jungHWANS** [comment]
y=B1 + B2*X1 + B3*X2 + ......

B1 : 주당 최저 배당액 * 주식 수
B2 : 실제 이행 여부 (신뢰도)의 회귀계수
X1: 재무제표 내 배당 성향 적용시 예상 지급액  [결국 이것도 당기 순이익 먼저 추정해야할 듯]

**Comment by changwoo-seo94** [comment]
당기순이익 추정이나 다른 변수 추가 없이 주당현금배당을 시계열로 단순회귀분석하는 게 더 간단하지 않겄음??
